src/net.c

```diff
@@ -599,6 +600,12 @@ coap_free_context(coap_context_t *context) {
 #ifndef WITHOUT_ASYNC
   coap_delete_all_async(context);
 #endif /* WITHOUT_ASYNC */
+  CACHE_ITER_SAFE(context->cache, cp, ctmp) {
```

Comment:
Please use `HASH_ITER` (see previous comment on this).

Reply:
Updated